transmart-rest-api/grails-app/services/org/transmartproject/rest/dataExport/ExportService.groovy

```diff
@@ -0,0 +1,170 @@
+package org.transmartproject.rest.dataExport
+
+import grails.transaction.Transactional
+import org.quartz.JobBuilder
+import org.quartz.JobDataMap
+import org.quartz.JobDetail
+import org.quartz.Scheduler
+import org.quartz.TriggerBuilder
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Component
+import org.transmartproject.core.exceptions.InvalidRequestException
+import org.transmartproject.db.job.AsyncJobCoreDb as AsyncJob
+import org.transmartproject.db.user.User
+
+import javax.transaction.NotSupportedException
+
+@Transactional
+@Component("restExportService")
+class ExportService {
+
+    @Autowired(required = false)
```

Comment:
There is an issue with autowiring for some classes - e.g. ObservationsBuilderTests in rest-api were failing with exception:  org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'restExportService': Unsatisfied dependency expressed through field 'quartzScheduler'; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean found for dependency [org.quartz.Scheduler]: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}